### ISSUE TITLE
Auto loot of purchase gadget

### ISSUE THREAD

**Issue by Feerko** [OnlyNoobDied/GadgetsMenu#434]
It's doesn't work for gadgets only. If I purchase a gadgets i don't receive them.

**Comment by OnlyNoobDied** [comment]
If you already have that gadget. It won't auto equip.

**Comment by Feerko** [comment]
@OnlyNoobDied How can i have already this gadgets them purchase it? :D

**Comment by OnlyNoobDied** [comment]
My bad. I tried, it works.

**Comment by OnlyNoobDied** [comment]
Did you received any message?

**Comment by Feerko** [comment]
Yes, bought message. I see now that's doesn't work for pets, moprhs, cloaks and emotes too. It's only works for effects, hats, banners and suits.

**Comment by OnlyNoobDied** [comment]
Send me the msg pls.

**Comment by OnlyNoobDied** [comment]
I tried all of them, and they works.

**Comment by Feerko** [comment]
can you join connect to my server? i will show you.

**Comment by Feerko** [comment]
derpmc.pl

**Comment by OnlyNoobDied** [comment]
I need your whole file. Including cosmetics.

**Comment by Feerko** [comment]
[GadgetsMenu.zip](https://github.com/OnlyNoobDied/GadgetsMenu/files/1551164/GadgetsMenu.zip)

**Comment by OnlyNoobDied** [comment]
@Feerko Sorry, i can't find any issue about auto loot as it works well in my testing server and other people doesn't have the same issue.
